bentoml/cli/bento.py

```diff
@@ -237,3 +245,112 @@ def retrieve(bento, target_dir):
         safe_retrieve(bento_service_bundle_path, target_dir)
 
         click.echo('Service %s artifact directory => %s' % (name, target_dir))
+
+    @cli.command(
+        help='Containerize given Bento into a Docker image',
+        short_help="Containerize given Bento into a Docker image",
+    )
+    @click.argument(
+        "bento", type=click.STRING, callback=parse_bento_tag_callback,
+    )
+    @click.option('--push', is_flag=True)
+    @click.option(
+        '--docker-repository',
+        help="Prepends specified Docker repository to image name.",
+    )
+    @click.option(
+        '-u', '--username', type=click.STRING, required=False,
+    )
+    @click.option(
+        '-p', '--password', type=click.STRING, required=False,
+    )
+    def containerize(bento, push, docker_repository, username, password):
+        """Containerize specified BentoService.
+
+        BENTO is the target BentoService to be containerized, referenced by its name
```

Comment:
This might be a bit inconsistent in the codebase right now, the `BENTO` parameter in CLI commands in `create_bento_service_cli` actually supports both `name:version` tag and a local file path to the saved bundle. we should probably also support that here.

Some users use the `BentoService#save_to_dir` method to skip BentoML's model registry and build their own model management workflow, for those users, they will need to run `containerize` by giving it the saved path

Reply:
support added :)